src/databricks/labs/ucx/azure/access.py

```diff
@@ -512,9 +512,20 @@ def create_access_connectors_for_storage_accounts(self) -> list[tuple[AccessConn
         thread_name = "Creating access connectors for storage accounts"
         results, errors = Threads.gather(thread_name, tasks)
         if len(errors) > 0:
+            logger.error("Error creating all access connectors. Deleting incomplete ones.")
```

Comment:
The code does not delete the incomplete access connectors, it deletes the once which were completed

Reply:
Done